pkg/kubelet/apis/deviceplugin/v1alpha/api.proto

```diff
@@ -96,6 +96,8 @@ message AllocateResponse {
 	repeated Mount mounts = 2;
 	// Devices for the container.
 	repeated DeviceSpec devices = 3;
+	// Annotations for the container to pass to the runtime.
```

Comment:
Maybe change this comment to "Container annotations to pass to the container runtime"?